typescript/infra/scripts/agents/utils.ts

```diff
@@ -32,6 +33,12 @@ export class AgentCli {
       switch (role) {
         case Role.Validator:
           for (const chain of this.agentConfig.contextChainNames[role]) {
+            // check if current chain is in provided set of chains
+            if (!this.chains?.includes(chain)) {
```

Comment:
is the intended behavior to default to not skipping any chains if `this.chains` is undefined? iiuc as written it will always require this.chains to be truthy, which feels cumbersome

Reply:
no that's an oversight from me, will fix